### DISCUSSION TITLE
Academic Reverification Required

### DISCUSSION THREAD

**Discussion by devdinho** [community/community#137663] [GitHub Education]
### Hi everyone,

My github @freitasanderson account has been suspended since August 22nd and I can't get help or support anywhere.

I believe that after this verification issue, my account was blocked:
![image](https://github.com/user-attachments/assets/ec0081b2-d0ae-43b9-9a81-22d3165fff97)

When I try to log into my account, I see this message.
![image](https://github.com/user-attachments/assets/dbe27d81-4a1a-4bdd-afd4-975255608a75)

Please, I really need help. My account is my work tool. In it I store most of my projects and Gists of useful information.

**Comment by alx-mp4** [comment]
Hi @freitasanderson1, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!

**Comment by Injamul2k24** [comment]
Hi @freitasanderson1 , I recommend you to re-verify with your current ID card of your institute. With better camera so that it is proper and clearly visible.
